### PR TITLE
Dev vadim

### PR DESCRIPTION
**Pull request by floccinauc**
TITAN implementation in Fuse - basic version that imports most of the functionality from the original PACCMANN code, and uses the original TITAN data

### CODE REVIEW

File: fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/pl_model.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+#log = logging.getLogger(__name__)
+import numpy as np
+import colorama
+colorama.init(autoreset=True)
+from colorama import Fore, Back, Style
+
+
+class AffinityPredictorModule(pl.LightningModule):
+    def __init__(self, 
+        #protein_padding_length:int,
```
Comment (YoelShoshan) [inline fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/pl_model.py:34]: remove comments

File: fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/scripts/run_simple_hp_grid_search.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
+import sys
```
Comment (YoelShoshan) [inline fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/scripts/run_simple_hp_grid_search.py:1]: both this and the session manager one are too tied with LSF, we should consider removing them

File: fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/scripts/update_optuna.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
+import os
```
Comment (YoelShoshan) [inline fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/scripts/update_optuna.py:1]: let's not include this file as well, we are working anyway on organizing those utilities into a separate repo

File: fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/pl_model.py (hunk 1 above)
Comment (floccinauc) [reply fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/pl_model.py:34]: Done

File: fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/scripts/update_optuna.py (hunk 3 above)
Comment (floccinauc) [reply fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/scripts/update_optuna.py:1]: Removed

File: fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/runner.py [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+SELECTED_CONFIG = 'train_bmmca_full_PPI.yaml'
+
+for name,func in hydra_resolvers.items():
+   OmegaConf.register_new_resolver(name, func)
+
+
+# logging.info(f'CONFIGS_DIR={CONFIGS_DIR}')
+# logging.info(f'SELECTED_CONFIG={SELECTED_CONFIG}')
+
+def session_manager_and_logger_setup(cfg:dict)->None:
```
Comment (YoelShoshan) [inline fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/runner.py:51]: I do wonder what should we do with this, as we've migrated into having session_manager in biofm.

File: fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/utils.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    s = str(s).strip().replace(' ', '_')
+    return re.sub(r'(?u)[^-\w.]', '@', s)
+
+
+def listify(x):
+    if not isinstance(x, list):
+        x = [x]
+    return x
+
+# def split_classes(pairs_tsv:str, ligands_tsv:str, targets_tsv:str,
```
Comment (YoelShoshan) [inline fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/utils.py:87]: delete

**Review by YoelShoshan — APPROVED** [review]
LGTM - if possible, I'd like Moshiko to review this as well

**Pull request merged**